### PR TITLE
fix: use correct time zone shift & end time for order filters

### PR DESCRIPTION
**Pull request by asvishnyakov**
## Description
This PR depends on https://virtocommerce.atlassian.net/browse/PT-14841 fix
### Jira-link:
https://virtocommerce.atlassian.net/browse/ST-5571
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.46.0-pr-893-d492-d492da9b.zip
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.46.0-pr-893-93bb-93bb31ca.zip

### CODE REVIEW

**Review by Andrew-Orlov — APPROVED** [review]

File: client-app/core/utilities/common/index.ts [hunk 1]
```diff
@@ -68,7 +68,10 @@ export function numberToShortString(num: number): string {
   return (num / sizes[index].value).toFixed(2).replace(/\.0+$|(\.[0-9]*[1-9])0+$/, "$1") + sizes[index].suffix;
 }
 
-// convert Date value to string with format 'yyyy-MM-dd'
+/**
+ * Сonvert Date value to string with format 'yyyy-MM-dd'
+ * @deprecated Use full ISO 8601 format instead.
```
Comment (NaMax66) [inline client-app/core/utilities/common/index.ts:73]: Which function developer should use now?

File: client-app/core/utilities/date/index.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export function addTimeToStartDate(dateOnly?: string): string | undefined {
+  if (dateOnly) {
+    return toDateObject(dateOnly).toISOString();
+  }
+}
+
+export function addTimeToEndDate(dateOnly?: string): string | undefined {
```
Comment (NaMax66) [inline client-app/core/utilities/date/index.ts:7]: addDays(dateOnly: string, days: number)
or
addOneDay(dateOnly)

File: client-app/core/utilities/date/index.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export function addTimeToStartDate(dateOnly?: string): string | undefined {
```
Comment (NaMax66) [inline client-app/core/utilities/date/index.ts:1]: add comment //only accept date in format 'yyyy-MM-dd'

**Review by NaMax66 — CHANGES_REQUESTED** [review]

File: client-app/core/utilities/date/index.ts (hunk 2 above)
Comment (asvishnyakov) [reply client-app/core/utilities/date/index.ts:7]: In both these functions we add "TXX:XX:XX.XXZ" to the date, that's why they named this way

File: client-app/core/utilities/date/index.ts (hunk 3 above)
Comment (asvishnyakov) [reply client-app/core/utilities/date/index.ts]: Done

File: client-app/core/utilities/common/index.ts (hunk 1 above)
Comment (asvishnyakov) [reply client-app/core/utilities/common/index.ts]: Done

File: client-app/core/utilities/date/index.ts (hunk 2 above)
Comment (asvishnyakov) [reply client-app/core/utilities/date/index.ts]: Renamed

**Review by NaMax66 — APPROVED** [review]

**Pull request merged**